packages/components/src/hooks/use-previous.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const usePrevious = value => {
```

Comment:
Can we rename this shareable hook to something that has a clearer definition?

Reply:
usePreviousState?